ci_framework/roles/os_must_gather/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure directories are present
+  ansible.builtin.file:
+    path: "{{ cifmw_repo_setup_basedir }}/{{ item }}"
+    state: directory
+  loop:
+    - logs
+
+- name: Run must-gather command
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell:
+    cmd: >-
+      oc adm must-gather --image {{ cifmw_os_must_gather_image }}
+      {% if cifmw_os_must_gather_timeout is defined %}
+      --timeout {{ cifmw_os_must_gather_timeout }}
+      {% endif %}
+      {% if cifmw_os_must_gather_host_network | bool %}
+      --host-network=true
+      {% endif %}
+      --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs > os_must_gather.log
+  args:
```

Comment:
if you use the "cmd" parameter, you don't need to use the "args" level - just append the chdir and related at the same level.